py3status/modules/net_rate.py

```diff
@@ -59,6 +48,19 @@ def __init__(self, *args, **kwargs):
             value - value (float)
             unit - unit (string)
         """
+        self.all_interfaces = True
+        self.cache_timeout = 2
+        self.devfile = '/proc/net/dev'
+        self.format = "{interface}: {total}"
+        self.format_no_connection = ''
+        self.hide_if_zero = False
+        self.interfaces = ''
+        self.interfaces_blacklist = 'lo'
+        self.precision = 1
+        self.color = False
+        self.threshold_bad = 1
+        self.threshold_degraded = 1024
+
```

Comment:
The defaults should stay where they were as it makes things easier to read and they are class defaults rather than instance ones.